.env.example

```diff
@@ -16,3 +16,6 @@ PAGE_FREEZER_API_KEY="API_KEY"
 export WEB_MONITORING_DB_STAGING_URL="https://api-staging.monitoring.envirodatagov.org"
 export WEB_MONITORING_DB_STAGING_EMAIL=""
 export WEB_MONITORING_DB_STAGING_PASSWORD=""
+
+# This is used in the web_monitoring.diffing_server module.
+export ACCESS_CONTROL_ALLOW_ORIGIN_HEADER="*"
```

Comment:
Please keep the newline at the end of the file!

Reply:
Intellij removes the newline at the end of the file and I have to remember to add two of them. Sorry about this!